app/helpers/application_helper.rb

```diff
@@ -16,6 +16,10 @@ def admin_or_moderator?
     access_denied
   end
 
+  def user_not_banned?
```

Comment:
change to positive method

e.g. user_banned? => current_user.banned? and that's all (don't forget to check if nil if needed)